Sources/Crypto/Digests/Digests_generated_file_do_not_edit.swift

```diff
@@ -15,6 +15,13 @@
 @_exported import CryptoKit
 #else
 
+// MARK: - Generated file, do NOT edit
+// any edits of this file WILL be overwritten and thus discarded
+// see section `GYB` in `README` for more details.
+
```

Comment:
I've no idea why gyb adds these three additional new lines... how can I rectify this? I'm using:
```bash
gyb --line-directive ""
```

Reply:
The three new lines are coming from the gaps in between the formatting directives. I suspect a previous run of swift-format cleared them up. Try deleting the empty lines between the directives and re-running.

Reply:
That is, the empty lines between the three sections of:

```
%{
 digests_and_length = [{"name": "SHA256", "count": 32},{"name": "SHA384","count":48},{"name":"SHA512", "count": 64},{"name":"SHA1", "count":20, "prefix":"Insecure"},{"name":"MD5", "count":16, "prefix":"Insecure"}]
 }%

% for HF in digests_and_length:
%{
    name = HF["name"]
    byteCount = HF["count"]
    wordsCount = (byteCount*8)/64 + (0 if ((byteCount*8)%64) == 0 else 1)
}%

%{
    if "prefix" in HF.keys():
        protocol_prefix = "extension " + HF["prefix"] + "{"
        protocol_suffix = "}"
    else:
        protocol_prefix = ""
        protocol_suffix = ""
}%
```